### PR TITLE
Make sure properties fit on screen. Add Drawcalls in top right corner.

### PR DESCRIPTION
**Pull request by AGulev**
This is how it looks like
image

### CODE REVIEW

File: engine/profiler/src/profile_render.cpp [hunk 1]
```diff
@@ -291,13 +291,11 @@ namespace dmProfileRender
         return Area(p, s);
     }
 
-    static Area GetPropertiesArea(DisplayMode display_mode, const Area& profiler_area, int counters_count, float scale)
```
Comment (AGulev) [inline engine/profiler/src/profile_render.cpp:294]: We don't really use scale anymore

File: engine/profiler/src/profile_render.cpp [hunk 2]
```diff
@@ -352,26 +350,104 @@ namespace dmProfileRender
...
         uint64_t max_frame_time     = render_profile->m_MaxFrameTime;
         uint64_t frame_time         = thread->m_SamplesTotalTime;
+        uint32_t drawcalls          = 0;
 
         float frame_time_f = frame_time / (float)ticks_per_second;
         float max_frame_time_f = max_frame_time / (float)ticks_per_second;
 
-        int l = dmSnPrintf(buffer, sizeof(buffer), "Frame: %6.3f Max: %6.3f", frame_time_f, max_frame_time_f);
+        {
+            // Properties
```
Comment (AGulev) [inline engine/profiler/src/profile_render.cpp:380]: Draw properties before the header to get Drawcalls count from properties

File: engine/profiler/src/profile_render.cpp [hunk 3]
```diff
@@ -352,26 +350,104 @@ namespace dmProfileRender
...
+
+                const char* name = dmHashReverseSafe32(property.m_NameHash);
+                if (name[0]=='r' && name[1]=='m' && name[2]=='t' && name[3]=='p' && name[4]=='_')
+                {
+                    name = name + 5;
+                }
+                params.m_Text = name;
+
+                if (name[0]=='D' && name[1]=='r' && name[2]=='a' && name[3]=='w' && name[4]=='C' && name[5]=='a' && name[6]=='l'
+                    && name[7]=='l' && name[8]=='s')
```
Comment (AGulev) [inline engine/profiler/src/profile_render.cpp:424]: ugly, but fast an simple
I think it's good enough for the profiler and a single use case

Comment (JCash) [inline engine/profiler/src/profile_render.cpp:424]: I prefer it like this:
```
if (strstr(name, "DrawCalls") == name)
```
In truth, the "rmtp_" case above could be written that way too, it's not performance critical code.

**Review by JCash — CHANGES_REQUESTED** [review]

File: engine/profiler/src/profile_render.cpp (hunk 3 above)
Comment (AGulev) [reply engine/profiler/src/profile_render.cpp]: thx, fixed

**Review by JCash — APPROVED** [review]
lgtm

**Pull request merged**